install/build_from_source.md

```diff
@@ -254,7 +254,7 @@ The user can set any CMake variable or option that controls the build process fr
 | qt | | Qt graphics backend, requires libqt >= 4.8 |
 | qtgsi | * | GSI's Qt integration, requires libqt >= 4.8 |
 | roofit | * | Build the libRooFit advanced fitting package |
-| root7 | OFF | ROOT 7 support ([read more](/root-7)) |
+| root7 | OFF | ROOT 7 support |
```

Comment:
can you please make this a link to the ROOT7 section in this file?

Reply:
There is none, right? @Axel-Naumann Did I get this wrong?

Reply:
https://root-project.github.io/web/install/build_from_source/#enabling-experimental-features-aka-root7

Reply:
This is just right above the table with the build option. I can add the link there, though it seems to me not necessary.

Reply:
And it's not "read more", it says just that you required C++14. If you still vote for adding it, I'm happy to do so!

Reply:
please :) it will contain more info (or links to more info) in the future

Reply:
Done! :)